### ISSUE TITLE
What is the meaning of MVRC_LOSS? What use is it？

### ISSUE THREAD

**Issue by Pter61** [jackroos/VL-BERT#63]
What is the meaning of MVRC_LOSS? What use is it？

**Comment by jackroos** [comment]
It means the loss of pre-training task Masked RoI Classification with Linguistic Clues (see Session3.3 "Task#2" in [our paper](https://arxiv.org/pdf/1908.08530.pdf) for details).
